samples/botframework-webchat-control/yarn.lock

```diff
@@ -0,0 +1,13127 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Do we need this file? I think we should not put this in the repository.

Reply:
Yes, it should be committed so that every contributor to ensure they have the same set of dependencies.